stix_shifter_modules/aws_athena/stix_translation/json/guardduty_from_stix_map.json

```diff
@@ -0,0 +1,79 @@
+{
+  "ipv4-addr": {
+    "fields": {
+      "value": ["eth0_private_ip", "eth1_private_ip", "public_ip", "remote_ip"],
+      "x_aws_interface_id": ["interface_id"],
+      "x_aws_remote_city_name": ["city_name"],
+      "x_aws_remote_country_name": ["country_name"]
+    }
+  },
+  "ipv6-addr": {
+    "fields": {
+      "value": ["ipv6_address"],
+      "x_aws_interface_id": ["interface_id"]
+    }
+  },
+  "network-traffic": {
+    "fields": {
+      "src_port": ["local_port"],
+      "dst_port": ["remote_port"],
+      "src_ref.value": ["eth0_private_ip"],
+      "dst_ref.value": ["remote_ip"],
+      "protocols[*]": ["protocol"]
+    }
+  },
+  "domain-name":{
+    "fields": {
+      "value": ["eth0_private_dns_name","eth1_private_dns_name", "public_dns_name", "dns_domain"]
+    }
+  },
+  "user-account": {
+    "fields": {
+      "user_id": ["principal_id"],
+      "account_login": ["user_name"]
+    }
+  },
+  "x-aws-details": {
+    "fields": {
+      "account_id": ["accountid"],
+      "region": ["region"]
+    }
+  },
+  "x-aws-instance": {
+    "fields": {
+      "instance_id": ["instance_id"],
+      "image_id": ["image_id"],
+      "availability_zone": ["availability_zone"]
+    }
+  },
+  "x-aws-vpc": {
+    "fields": {
+      "vpc_id": ["vpc_id"],
+      "subnet_id": ["subnet_id"],
+      "security_group_name": ["security_group_name"],
+      "security_group_id": ["security_group_id"]
+    }
+  },
+  "x-aws-api": {
+    "fields": {
+      "access_key_id": ["access_key_id"],
+      "api": ["api"],
+      "api_service_name": ["api_service_name"]
+    }
+  },
+ "x-ibm-finding": {
+  "fields": {
+    "name": ["title"],
+    "finding_id": ["id"],
```

Comment:
I don't think we need to have an id on the finding object. Where would it be coming from? The other connectors that use the finding object don't specify an id.

Reply:
@delliott90 The finding id is now removed from the mapping and code is updated in PR branch